Casks/s/studiolinkstandalone.rb

```diff
@@ -19,9 +19,11 @@
   homepage "https://studio-link.de/"
 
   livecheck do
-    url "https://gitlab.com/studio.link/app.git"
-    regex(/^v?(\d+(?:\.\d+)*)[._-]stable$/i)
+    url "https://doku.studio-link.de/standalone/installation-standalone.html"
+    regex(/Version:\s+v?(\d+(?:\.\d+)+)[._-]stable/i)
```

Comment:
```suggestion
    regex(%r{href=.*?/releases/v?(\d+(?:\.\d+)+)-stable/macos}i)
```

The page contains a link to the zip file we use in the cask, so it would be better to match the version from the URLs instead of the loose `Neue Version: v21.07.0-stable (06.07.2021)` text.

We could expand this to `regex(%r{href=.*?/releases/v?(\d+(?:\.\d+)+)-stable/macos[^/]*/signed/}i)`, if we want to be more specific. That would only match the signed zip file that we use in the cask but the check may break in the future if upstream changes the setup (e.g., if they promote the signed version to be the default and drop the `/signed/` directory structure). I think we can start with the looser approached in the suggestion above and switch to the more specific regex if/when a need arises in the future.